cnf/ext/repositories.bnd

```diff
@@ -37,6 +37,11 @@ mavencentral:           https://repo.maven.apache.org/maven2
         locations="https://dl.bintray.com/bndtools/eclipse-repo/4.10/index.xml.gz";\
         poll.time=-1;\
         cache="${workspace}/cnf/cache/stable/Eclipse-2018-12",\
+    aQute.bnd.repository.osgi.OSGiRepository;\
```

Comment:
Obviously we don't want to commit this. Are there any artifacts missing from the bintray repo I need to add?

Reply:
Yes, there are arfiacts missing from the slim repo, see the diff. I removed the repo reference.